atlas/commands/bootstrap/bootstrap.go

```diff
@@ -194,3 +203,12 @@ func initRepo() error {
 	fmt.Println("done!")
 	return nil
 }
+
+func initHelm(app application.Application) error {
```

Comment:
Should we add common targets into make files to build helm chart archive ?

Reply:
I mean in Makefile template

Reply:
I agree on the common make target. There may already be an emerging make target for the chart in our repos. 

For inspiration on how the make targets could work checkout kubebuilder. 

`make install` will deploy the application into a running k8s cluster. I would expect something similar to happen here.